### PR TITLE
 peers: handle maxAge option

### PR DESCRIPTION
**Pull request by robertkowalski**
handle option maxAge to remove stale peers from peercache

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ If `opts` is specified, then the default options (shown below) will be overridde
   concurrency: 16, // k-rpc option to specify maximum concurrent UDP requests allowed (Number, 16 by default)
   hash: Function,  // custom hash function to use (Function, SHA1 by default),
   krpc: krpc()     // optional k-rpc instance
+  timeBucketOutdated: 900000 // check buckets every 15min
+  maxAge: Infinity  // optional setting for announced peers to time out
```
Comment (mafintosh) [inline README.md]: could you some missing `,` above? :)

**Comment by mafintosh** [comment]
It is failing because of standard, I'll fix that.

**Comment by robertkowalski** [comment]
ugh sorry, my bad

**Comment by mafintosh** [comment]
7.9.0

**Pull request merged**